SauceDemo.Tests/LoginTests.cs

```diff
@@ -0,0 +1,100 @@
+﻿using OpenQA.Selenium;
+using Xunit.Abstractions;
+using SauceDemo.Core.Driver;
+using log4net;
+using FluentAssertions;
+using SauceDemo.Business.Steps;
+
+namespace SauceDemo.Tests
+{
+    [Collection("Global logger collection")]
+    public class LoginTests : IDisposable
+    {
+        private readonly IWebDriver driver;
+        private LoginPageSteps loginPageSteps;
+        private static readonly ILog log = LogManager.GetLogger(typeof(LoginTests));
+
+        public LoginTests(ITestOutputHelper output)
+        {
+            log.Info("Initializing test...");
+            driver = WebDriverFactory.CreateWebDriver("Edge");
+            driver.Manage().Window.Maximize();
+            loginPageSteps = new LoginPageSteps(driver);
+            log.Info("Go to Login Page");
+            loginPageSteps.GoToLoginPage();
+        }
+
+        public static IEnumerable<object[]> ValidUsernames()
+        {
+            yield return new object[] { "standard_user" };
+            yield return new object[] { "locked_out_user" };
+            yield return new object[] { "problem_user" };
+            yield return new object[] { "performance_glitch_user" };
+            yield return new object[] { "error_user" };
+            yield return new object[] { "visual_user" };
+        }
+
+        [Fact(DisplayName = "UC-1 - Test Login Form with Empty Credentials")]
+        public void UC1_EmptyCredentials_ShouldShowUsernameRequired()
+        {
+            #region Parameters
+
+            var invalidUsernameAndPassword = "temp";
+
+            #endregion
+
+            loginPageSteps.EnterUsernameAndPassword(invalidUsernameAndPassword, invalidUsernameAndPassword);
```

Comment:
it would be fine if you will write
loginPage.EnterUsername
loginPage.EnterPassword